ordering/__init__.py

```diff
@@ -114,8 +123,8 @@ def __init__(self, ordering: Ordering[T], item: T) -> None:
 
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, OrderingItem):
-            return bool(self.item == other)
```

Comment:
`mypy --strict` complains about the typing of this line without the `bool` wrapping.